recipes/signac-dashboard/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "signac-dashboard" %}
+{% set version = "0.1.0" %}
+{% set sha256 = "bb4214e7f2be4185f7e0e1550f544b2b44f6ad04e426fedec754cf8fc9dae0a3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://bitbucket.org/glotzer/signac-dashboard/get/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [py<34]
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - signac
+    - flask
+    - flask-assets
+    - flask-cache
+    - flask-turbolinks
+    - libsass
+    - jsmin
+    - cssmin
+
+test:
+  imports:
+    - signac_dashboard
+
+about:
+  home: https://bitbucket.org/glotzer/signac-dashboard
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: LICENSE.txt
+  summary: 'Search and browse through signac data spaces in the browser.'
+
+  description: |
+    Data visualization, analysis, and "dashboard" monitoring tool
+    as part of the signac framework. The signac-dashboard interface
+    allows users to rapidly view data managed in a signac project.
+  doc_url: https://bitbucket.org/glotzer/signac-dashboard/overview
+  dev_url: https://bitbucket.org/glotzer/signac-dashboard/src
+
+extra:
+  recipe-maintainers:
+    - csadorf
+    - bdice
```

Comment:
@bdice, can you confirm that you're happy to [be a maintainer](http://conda-forge.readthedocs.io/en/latest/recipe.html#maintainer-role) for the `signac-dashboard` conda-forge package?